src/views/ResourcesSupport/AllResources/CveServices.vue

```diff
@@ -485,6 +480,20 @@
                 </li>
                 <li><a href="https://cveproject.github.io/cve-schema/schema/v5.0/docs/" target="_blank">Schema Documentation</a></li>
               </ul>
+              <p>CVE Bulk Download files in CVE JSON 5.0 format:</p>
+              <ul>
+                <li><a href="/Downloads">Downloads page on CVE.ORG</a></li>
+                <li><a href="" target="_blank">cvelistV5 repository on GitHub</a></li>
```

Comment:
Recommended code to user <router-link> for in-app links and fix erroneous 'cvelistV5 repository on GitHub' link:
```
<li><router-link to="/Downloads">Downloads page on CVE.ORG</router-link></li>
<li><a href="https://github.com/CVEProject/cvelistV5" target="_blank">cvelistV5 repository on GitHub</a></li>
```